scrapers/AbbyWinters.yml

```diff
@@ -0,0 +1,28 @@
+name: Abby Winters
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.abbywinters.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Studio:
+      Performers:
+        Name:
+          selector: //td/a[starts-with(@href,'https://www.abbywinters.com/nude_girl/')]/text()
+      Title:
+        selector: //div[starts-with(@class,'page-heading')]/div/h1/text()
+      Details:
+        selector: //div[@class='description padd-bottom-15 padd-left-15 padd-right-15']/p/span/text()
+      Tags:
+        Name:
+          selector: //p/a[starts-with(@href,'https://www.abbywinters.com/fetish/')]/text()|//p/a[starts-with(@href,'/browse/')]/text()|//p/a[starts-with(@href,'/amateurs/')]/text()
+      Image:
+        selector: //div[@class='video-player-container video-responsive']/@data-poster
+
+driver:
+  useCDP: true
+debug:
+  printHTML: true
```

Comment:
This scraper doesn't need CDP to function so this should be taken off, as well as the debug line for the final product.